source/modules/configuration/crontasks.rst

```diff
@@ -15,14 +15,18 @@ Configure CLI mode
 To run CLI mode tasks, you will need to configure your external task scheduler to call GLPI's `front/cron.php` file periodically.
 It is recommended to have it run every minute to ensure that actions that are ready to run, get ran as soon as possible.
 
-For Linux/MacOS, you should add the following to the web server's user's (www-data, apache, etc) crontab:
-
 ::
 
    * * * * * php GLPI/front/cron.php
 
+For Linux/MacOS, you should add the following to the web server's user's (**www-data, apache, etc**) crontab:
+
+::
+
+   * * * * * www-data cd GLPI/front && php cron.php
```

Comment:
Either you use the user crontab, and you do not to specify user name, or you use the global crontab and you must add the user name to use for run. 
Seems inconsistent here.

Reply:
It may be OK to document both options separately as some user may want to manage all "system" crontabs together and not use the web server user's crontab.

I don't see the functional difference of changing the actual command from "php GLPI/front/cron.php" to "cd GLPI/front && php cron.php".